Koha/Schema/Result/InvoiceAdjustment.pm

```diff
@@ -0,0 +1,155 @@
+use utf8;
+package Koha::Schema::Result::InvoiceAdjustment;
+
+# Created by DBIx::Class::Schema::Loader
+# DO NOT MODIFY THE FIRST PART OF THIS FILE
+
+=head1 NAME
+
+Koha::Schema::Result::InvoiceAdjustment
+
+=cut
+
+use strict;
+use warnings;
+
+use base 'DBIx::Class::Core';
+
+=head1 TABLE: C<invoice_adjustments>
+
+=cut
+
+__PACKAGE__->table("invoice_adjustments");
+
+=head1 ACCESSORS
+
+=head2 adjustment_id
```

Comment:
Never understood why koha insists on table_id format.. dbic recommends (and I agree) with just 'id' for the field name.. you already have the table in the calling construct usually `adjustment->id` vs `adjustment->adjustment_id`